src/api/spec/models/project_spec.rb

```diff
@@ -326,4 +326,103 @@
       end
     end
   end
+
+  describe '#save_distributions' do
+    let(:factory) { create(:project, name: 'openSUSE:Factory') }
+    let!(:factory_distribution) { create(:distribution, project: 'openSUSE:Factory', name: 'openSUSE Tumbleweed', reponame: 'openSUSE_Tumbleweed') }
+    let!(:factory_repository) { create(:repository, project: factory, name: 'standard') }
+    let(:method) { project.save_distributions(['openSUSE_Tumbleweed']) }
```

Comment:
Can you change that `method` name? I would prefer to call it `project_save_distributions`